tests/ha/test_ha_pod_restart.py

```diff
@@ -2959,13 +2960,23 @@ def test_continuous_reads_during_pod_restart_ros(self):
         assert_utils.assert_true(resp[0], resp)
         LOGGER.info("Step 8: Cluster is in good state. All the services are up and running")
         event.clear()
+        LOGGER.debug("Event is cleared")
         thread.join()
-
+        LOGGER.debug("Thread has joined.")
         LOGGER.info("Verifying responses from background process")
         responses = {}
         while len(responses) != 2:
             responses = output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        if not responses:
+            assert_utils.assert_true(False, "Background S3bench Failures")
+        LOGGER.debug("Background S3bench responses : %s", responses)
+        if not responses["pass_res"]:
+            assert_utils.assert_true(False,
+                                     "No Background S3bench logs when event was cleared")
         nonbkgrd_logs = list(x[1] for x in responses["pass_res"])
+        if not responses["fail_res"]:
```

Comment:
so fail_res gives logs for IOs run before foreground ops?

Reply:
> so fail_res gives logs for IOs run before foreground ops?

It has logs for IOs while event was set.